src/lists.py

```diff
@@ -42,17 +42,27 @@ def print_list(x):
 match sys.argv[1]:
     case "mean":
         # put your solution to the first exercise here
-        mean = "mean of x"
+        sum = 0 
+        for i in x: 
+            sum=sum+i
+        mean = sum/len(x)
         print(mean)
 
     case "times":
         # Put your solution to the second exercise here
         times_three = []
+        for i in x:
+            times_three.append(i*3)
         print_list(times_three)
 
     case "even":
         # Put your solution to the third exercise here
         even = []
+        for i in x:
+            if i % 2 == 0:
+                even.append(i)
+            else:
+                continue
```

Comment:
This code isn't necessary. You do the `if` part whenever the condition is true, and the `else` part does nothing, so you could leave it out completely.